interface/grpc/core/service.proto

```diff
@@ -1,10 +1,10 @@
 syntax = "proto3";
 
-package service;
+package core;
 
 // This is the definition of a MESG Service.
 message Service {
-  string id = 10;                           // Service's unique id hash.
+  string id = 10;                           // Service's unique id service hash.
```

Comment:
Can't we use `ID` here and like that we only have `ID` in the codebase and not `Id` and `ID`

Reply:
done